src/workload.py

```diff
@@ -164,3 +165,30 @@ def generate_password(self) -> str:
             String of 32 randomized letter+digit characters
         """
         return "".join([secrets.choice(string.ascii_letters + string.digits) for _ in range(32)])
+
+    @override
+    def get_version(self) -> str:
+
+        if not self.healthy:
+            return ""
+
+        stat = [
+            f"echo 'stat' | (exec 3<>/dev/tcp/localhost/{CLIENT_PORT}; cat >&3; cat <&3; exec 3<&-)"
+        ]
+
+        # timeout needed as it can sometimes hang forever if there's a problem
+        # for example when the endpoint is unreachable
+        timeout = ["timeout", "10s", "bash", "-c"]
```

Comment:
**out of scope:** This also happens somewhere else. Maybe it's worth to make the timeout part of the exec API

Reply:
Removed in 9888223f8df7bdcf93439c772099c8d2a74d0b7f. The timeout is already in `.healthy`